requirements_test.txt

```diff
@@ -0,0 +1,3 @@
+-r requirements.txt
+pep8
+-e "git+https://github.com/magfest/sideboard#egg=sideboard"
```

Comment:
`install_requires` in `setup.py` can't handle all same formats as `pip` (like this `git+sideboard` repo). Since we're populating `install_requires` by reading `requirements.txt`, I had to move the `git+sideboard` line into a separate file, `requirements_test.txt`.

Some good reading on this here:
* https://packaging.python.org/requirements/
* https://caremad.io/posts/2013/07/setup-vs-requirement/